op-chain-ops/script/script_test.go

```diff
@@ -0,0 +1,32 @@
+package script
+
+import (
+	"testing"
+
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+)
+
+func TestScript(t *testing.T) {
+	logger, captLog := testlog.CaptureLogger(t, log.LevelInfo)
+	af := foundry.OpenArtifactsDir("../../packages/contracts-bedrock/forge-artifacts")
```

Comment:
It could be good to not couple so tightly to the contracts artifacts here so that we can reduce deps on the monorepo structure, this will make it harder to do something like remove the contracts package from the monorepo in the future

Reply:
done